tools/bug-report/pkg/content/content.go

```diff
@@ -154,18 +140,43 @@ func GetIstiodInfo(p *Params) (map[string]string, error) {
 		return nil, fmt.Errorf("getIstiodInfo requires namespace and pod")
 	}
 	ret := make(map[string]string)
-	for _, url := range istiodURLs {
-
-		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, "discovery", fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
+	for _, url := range common.IstiodDebugURLs(p.ClusterVersion) {
+		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.DiscoveryContainerName, fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
 		if err != nil {
 			return nil, err
 		}
-
 		ret[url] = out
 	}
 	return ret, nil
 }
 
+// GetNetstat returns netstat for the given container.
+func GetNetstat(p *Params) (map[string]string, error) {
+	if p.Namespace == "" || p.Pod == "" {
+		return nil, fmt.Errorf("getNetstat requires namespace and pod")
+	}
+
+	out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.ProxyContainerName, "netstat -natpw", p.DryRun)
+	if err != nil {
+		return nil, err
+	}
+	return retMap("netstat", out, err)
+}
+
+// GetNetfilter returns netfilter for the given container.
+/*func GetNetfilter(p *Params) (map[string]string, error) {
```

Comment:
remove dead code

Reply:
it's not dead, just sleeping :-) i absolutely cannot figure out the right combination of escapes to pass a for loop to exec (either through os.exec or using pkg/kube).